src/lib.rs

```diff
@@ -979,14 +1105,18 @@ impl<T: Clone + Integer + Signed> Signed for Ratio<T> {
 }
 
 // String conversions
-impl<T> fmt::Display for Ratio<T>
+impl<T: Clone + Integer> fmt::Display for Ratio<T>
 where
-    T: fmt::Display + Eq + One,
+    T: CheckedMul + fmt::Display,
```

Comment:
I missed the fact that you altered `Display` -- this is a breaking change.

Reply:
Yes. I couldn't think of an integer type that doesn't implement this trait.
Potentially, if that's a viable use case, then we could remove the change for the Display implementation and simply leave the stand-alone function for formatting as decimal. However, to me that looks less useful.

Reply:
I prefer to stick to the letter of the law when it comes to breaking changes.  A plausible case would be if someone had a newtype integer of their own (wrapping an existing integer type), where they hadn't bothered to implement `CheckedMul` since they weren't using that.

Reply:
I thought that still would be with accordance to semver since we didn't hit the `1.0` yet, but fair enough, I'll amend the PR.

Reply:
Ok, it's done